### ISSUE TITLE
074086

### ISSUE THREAD

**Issue by d4rios** [baskerville/bspwm#1352]
Currently in bspwm I usually have a single Polybar located at the top of the screen, which I modify from time to time. Yesterday, it occurred to me to try a different Polybar and place it at the bottom of the screen and I noticed the following.

When changing from one bar to another, Polybar remembers the space occupied by the previous one leaving me more margin above or below, depending on the situation of the previous bar on the screen. What the hell! Because it doesn't leave me space proportionally on both sides (top and bottom?). Does anyone know how to fix this? Thank you all in advance. 

bspwm -v
0.9.10

polybar -v
polybar 3.5.7

In Polybar --> margin-bottom = 0 --> margin-top = 0

bspc config window_gap
10

![imagen](https://user-images.githubusercontent.com/80931135/149009008-1d1ece1f-3b0f-4717-988f-34deacec1191.png)

![imagen](https://user-images.githubusercontent.com/80931135/149009060-6e56e0a5-e47b-4698-9b83-4017afd37319.png)

![imagen](https://user-images.githubusercontent.com/80931135/149009143-6914664e-33b0-4b93-9ee8-aa30bff528e3.png)

**Comment by ServoAtya** [comment]
Yeah, i have the same issue, really annoying sometimes, for me only a problem this, if my monitor going to blank, and after wake up i have extra spacing, only logout fixes this fully.

**Comment by hohiowo** [comment]
i think you can work around this by putting `override-redirect = true` in your polybar config file, and then changing bspwm's margins using `bspc config top_padding/bottom_padding <your bar's height in px>` each time you change your theme/update your bar
